### DISCUSSION TITLE
Remove or invert necessity for the Audio_in OSC to have a step set.

### DISCUSSION THREAD

**Discussion by BKLronin** [bangcorrupt/hacktribe#252] [Ideas]
Thanks to midi though iḿ currently using hacktribe as center of the hardware. As you know when recording in a full legth track from external using the resample function you have to set the appropiate steps to get recorded. That is fine if you have recorded the actual midi notes before but when recording like a guitear or not midi connected synth etc you always have to set the first step and let it tie to the end completely to not have choppy sound.

Is there maybe a possibility to invert or bypass the logic for this OSC? Like always on, no matter if step? That would make sampling of complete steps and reduce possbile sources of failure / clicks and pops due too open and closing gates too early etc.

What do you think?  Maybe there are other recording techniques Iḿ not aware of currently.

**Comment by bangcorrupt** [comment]
I think this should be possible at some point.  A workaround could be to have a template pattern with parts to copy, one with the gate open for the full sequence.

**Reply by BKLronin** [reply]
Thats a good idea, I´ll probably make 7 and 8 the default full gate tracks.
